### ISSUE TITLE
Compatibility for Weewx >= 4.10

### ISSUE THREAD

**Issue by Daveiano** [Daveiano/weewx-wdc#111]
### Discussed in https://github.com/Daveiano/weewx-wdc/discussions/110

<div type='discussions-op-text'>

<sup>Originally posted by **JonFear** February  1, 2023</sup>
Hi!

Your favorite pain in the A**.

Applied the update from Weewx 4.9.1 to 4.10.1 and the skin has stopped working. I have Belchertown running in parallel and it is still working.

Any thoughts? Is there any log files I can copy for review?

Thanks

Jon </div>

- [x] Update weewx version comparison 
- [x] Update delta times, see weewx/weewx/pull/807

**Comment by Daveiano** [comment]
The version comparison was removed because a simple string comparison is too flasky:

```
if weewx.__version__ < "4.6":
    raise weewx.UnsupportedFeature(
        "weewx 4.6 and newer is required, found %s" % weewx.__version__
    )
```

"4.10.0" is "smaller" than "4.6". This would need a more complex solution or the usage of a third-party package which is not the scope of the skin.

**Comment by Daveiano** [comment]
Included in v2.3.3
